cmake/qtifw/install_registerfiletype.qs

```diff
@@ -46,12 +49,13 @@ function Component()
       // Note: Here you do **not** need to specify the specific iconID, since the EP-launch icon id is actually zero, which is default
       //  eg: var iconId = 0; icon = "@TargetDir@\\EP-Launch.exe," + iconId
       // EP-Launch.exe
-      component.addElevatedOperation("RegisterFileType", "idf", "@TargetDir@\\EP-Launch.exe %1", "EnergyPlus Input Data File", "text/plain");
-      component.addElevatedOperation("RegisterFileType", "imf", "@TargetDir@\\EP-Launch.exe %1", "EnergyPlus Input Macro File", "text/plain");
-      component.addElevatedOperation("RegisterFileType", "epg", "@TargetDir@\\EP-Launch.exe %1", "EnergyPlus Group File", "text/plain");
+      component.addElevatedOperation("RegisterFileType", "idf", `${targetDir}\\EP-Launch.exe %1`, "EnergyPlus Input Data File", "text/plain");
+      component.addElevatedOperation("RegisterFileType", "imf", `${targetDir}\\EP-Launch.exe %1`, "EnergyPlus Input Macro File", "text/plain");
+      component.addElevatedOperation("RegisterFileType", "epg", `${targetDir}\\EP-Launch.exe %1`, "EnergyPlus Group File", "text/plain");
```

Comment:
I didn't realize you could use backticks in this way here, but I trust since this is fixing the issues.

Reply:
Javascript is not my forte, but I brushed up on it a bit recently. This is called a template string/litteral: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals